<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QDialog" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator Control Panel</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>731</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Manage Operators</string>
    </attribute>
    <widget class="QTableView" name="operatorsView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>681</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Registered Operators</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDisableOperator">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>181</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Operator(s)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblOperatorsCount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>54</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnableOperator">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>181</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Operator(s)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnResetPassword">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>290</y>
       <width>181</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Password</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Location Settings</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>631</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Local Government:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Registration Area:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Registration Centre:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblCurrentState">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblCurrentLga">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblCurrentRa">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblCurrentPu">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Change Location Settings</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>631</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="inpState">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Local Government:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="inpLga">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Registration Area:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="inpRa">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Registration Centre:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="inpPu">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Update OpenVR</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Update OpenVR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>NOTE: Please connect the Patching Removable Disk to the DDC Laptop</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>System Settings</string>
    </attribute>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>227</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>461</width>
       <height>115</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Fingerprint Reader Sensitivity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fpSensitivity"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="updateSettingsButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Maintenance</string>
    </attribute>
    <widget class="QPushButton" name="btnRepairDatabase">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Repair Database</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>operatorsView</tabstop>
  <tabstop>btnDisableOperator</tabstop>
  <tabstop>btnEnableOperator</tabstop>
  <tabstop>btnResetPassword</tabstop>
  <tabstop>inpState</tabstop>
  <tabstop>inpLga</tabstop>
  <tabstop>inpRa</tabstop>
  <tabstop>inpPu</tabstop>
  <tabstop>fpSensitivity</tabstop>
  <tabstop>updateSettingsButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ControlPanel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ControlPanel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
